README.md

```diff
@@ -1 +1,180 @@
-# budget-app
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+ 
+  <br/>
+
+  <h1><b>Budget App, Capstone Project</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Budget-app] <a name="about-project"></a>
+
+**[Budget-app]** is a blog web application. User can create accounts, posts and users can add likes and comments for those posts. Built with ruby on rails.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.sqlite.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[sign in and sign up]**
+- **[form validations]**
+- **[Categorized and Transactioned]**
+- **[Rspec uni-testing]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo">https://budgetsapp.onrender.com/</a>
```

Comment:
- [ OPTIONAL ] I suggest you remove the link in front of the `live demo` highlighted in red in the image below and leave just this line highlighted in blue. I feel this would make your documentation more professional

![Screenshot 2023-04-23 at 00 33 21 (2)](https://user-images.githubusercontent.com/64774936/233811972-664a5ebd-5a62-4f63-b82b-510d79197577.png)

Reply:
- [ ] [OPTIONAL] Great job done @akhror-valiev 👍🏽 , however, In the `Acknowledgement` section of your project, I will recommend that you give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) and the project requirement.